subitem.go

```diff
@@ -47,30 +52,51 @@ type SubscriptionItemBillingThresholdsParams struct {
 // SubscriptionItemListParams is the set of parameters that can be used when listing invoice items.
 // For more details see https://stripe.com/docs/api#list_invoiceitems.
 type SubscriptionItemListParams struct {
-	ListParams   `form:"*"`
-	Subscription *string `form:"subscription"`
+	EndingBefore  *string   `form:"ending_before"`
+	Expand        []*string `form:"expand"`
+	Limit         *int64    `form:"limit"`
+	StartingAfter *string   `form:"starting_after"`
+	Subscription  *string   `form:"subscription"`
+}
+type SubscriptionItemUsageRecordSummariesParams struct {
```

Comment:
What is this class about?

Reply:
I've removed it -- it didn't belong here.

Long story short, this should be `UsageRecordSummaryListParams` in a different file `usagerecordsummary.go`.